source/agora/consensus/EnrollmentManager.d

```diff
@@ -1464,22 +1464,22 @@ unittest
 
         PreImageInfo preimage = { utxo : utxos[idx],
             distance : cast(ushort)params.ValidatorCycle,
-            hash : cache[$ - params.ValidatorCycle - 1] };
+            hash : cache[$ - (params.ValidatorCycle + 1) - 1] };
```

Comment:
```suggestion
            hash : cache[$ - params.ValidatorCycle] };
```

Reply:
My intention was to make it clearer that we are taking the pre-image from offset `cycle` + 1. 
This will change back to  `hash : cache[$ - params.ValidatorCycle - 1] };`
in the other PR.

Reply:
If it changes back in the next PR, I don't really understand the point of this ?

Reply:
The the hashes will remain the same which is the point of this PR.